### PR TITLE
Add detailed installation instructions

### PR DESCRIPTION
**Pull request by sophokles73**
The instructions are mostly copied from the Hono project.

### CODE REVIEW

File: charts/hono/README.md [hunk 1]
```diff
@@ -1,36 +1,132 @@
 # Eclipse Hono
 
-## Introduction
-
 [Eclipse Hono™](https://www.eclipse.org/hono/) provides remote service interfaces for connecting large
 numbers of IoT devices to a back end and interacting with them in a uniform way regardless of the device
 communication protocol.
 
+This repository contains a *chart* that can be used to install Hono to a Kubernets cluster using the
```
Comment (calohmn) [inline charts/hono/README.md]: typo: Kubernetes

File: charts/hono/README.md [hunk 2]
```diff
@@ -39,28 +135,201 @@ The `values.yaml` file contains all possible configuration values along with doc
...
+
+  # mount (existing) Kubernetes secret which contains
+  # credentials for connecting to AMQP network
+  extraSecretMounts:
+  - amqpNetwork:
+      secretName: "my-secret"
+      mountPath: "/etc/custom"
+
+  # provide connection params
+  # assuming that "my-secret" contains an "amqp-credenials.properties" file
```
Comment (calohmn) [inline charts/hono/README.md]: typo: credentials

**Comment by sophokles73** [comment]
> How about adding the "Using Jaeger Tracing" chapter as well:

I deliberately left out all customizations that require compiling from source as these cannot be done by just setting configuration properties. i would still keep them in the Hono deployment docs, though.

**Comment by sophokles73** [comment]
@calohmn can you take another look?

**Pull request merged**